### PR TITLE
CXF 92626: Adding FCR 2 Metal connection Example

### PR DESCRIPTION
**Pull request by srushti-patl**
- Added zside_ap_type = METAL_NETWORK in cloud-router connection sub module
- Added FCR 2 Metal Connection example
- Removed aside_ap_type variable from cloud-router-connection module and cloud router connectivity examples

Note: I will update FCR-2-metal example once we have new version of terraform provider with Redundant Group Bug fix.

### CODE REVIEW

File: examples/cloud-router-2-metal-nimf-connection/versions.tf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.5.4"
+  required_providers {
+    equinix = {
+      source  = "equinix/equinix"
+      version = ">= 1.34.0"
```
Comment (thogarty) [inline examples/cloud-router-2-metal-nimf-connection/versions.tf:6]: Update version to v1.36.4. That's the version with the redundancy fix.

**Review by thogarty — CHANGES_REQUESTED** [review]
I'm also confused about the port2Azure example value addition. Is there a variable for that and it was just omitted from the .tfvars.example file?

**Comment by srushti-patl** [comment]
> I'm also confused about the port2Azure example value addition. Is there a variable for that and it was just omitted from the .tfvars.example file?

It was omitted from the tfvars.example file.
I have also added that variable to .tfvars.example file in the following examples:
Port-2-Azure-connection
Port-2-Azure-Redundant-connection
FCR-2-Azure-connection
FCr-2-Azure-Redundant-connection

**Review by thogarty — APPROVED** [review]
LGTM. Well done.

**Pull request merged**